test/elixir_analyzer/exercise_test/common_checks/function_capture_test.exs

```diff
@@ -48,6 +48,7 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.FunctionCaptureTest do
             |> Enum.map(fn x -> <<x>> end)
             |> Enum.reduce([], fn a, b, c -> {a, b, c} end)
             |> Enum.reduce([], &{&1, &2, &2})
+            |> Enum.map(fn -> %{} end)
```

Comment:
I was trying to find ways to have some kind of valid `fn ??? -> %{???}` notation, but I couldn't find any.